includes/migrations/class-wc-stripe-ocs-ap-default-on-update.php

```
@@ -0,0 +1,216 @@
+
+		// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_var_export -- distinguishing `false` (option unset) from string versions in the log message.
+		WC_Stripe_Logger::info( sprintf( '[OCS+AP 10.8] Migration started. previous_version=%s.', var_export( $previous_version, true ) ) );
+
+		if ( false === $previous_version ) {
+			WC_Stripe_Logger::info( '[OCS+AP 10.8] Skipping: new install (no previous_version recorded). Frontbook 10.8 path will be handled at OAuth time.' );
+			update_option( self::MIGRATION_FLAG_OPTION, 'yes' );
+			return;
+		}
+
+		if ( version_compare( (string) $previous_version, '10.8.0', '>=' ) ) {
+			WC_Stripe_Logger::info( sprintf( '[OCS+AP 10.8] Skipping: previous_version=%s already >= 10.8.0.', $previous_version ) );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'maybe_run'. (undefined)

(StaticAccess)